src/components/HelloWorld.vue

```diff
@@ -0,0 +1,129 @@
+<template>
```

Comment:
Do we need this file?

Reply:
no must've snuck in when i installed vuetify. oops